### DISCUSSION TITLE
2018 Captur / Alarm Perimetric/  UNDERHOOD_CF ?

### DISCUSSION THREAD

**Discussion by xspawn86** [cedricp/ddt4all#928] [Q&A]
What does "UNDERHOOD_CF" under the "Alarm Perimetric" menu mean and what does it do? Should it be "TRUE"? I tried it a bit, I wonder if I opened it by mistake?   
![bonnetalarm3](https://github.com/cedricp/ddt4all/assets/149533962/e03e1f8c-0249-41c1-8e14-f137a099874d)

**Comment by AndyIreland** [comment]
I think it is for if there is a switch under the hood (bonnet) - to do with the add on alarm system

**Reply by xspawn86** [reply]
thanks for reply, i solved. If you activate perimetric_alarm and underhood_cf together, flashers turn on when the door is locked and the hood is opened, but theres no horn.

**Reply by AndyIreland** [reply]
Have you adjusted this to 'true' (on your particular car ECU) ? - If so it might get your horn sounding as well when  the doors are locked but the window smashed and door latch opened from inside the car 
![image](https://github.com/cedricp/ddt4all/assets/49172217/bf5c855d-304f-4624-861f-a2aa90cfe70d)

**Reply by ntgnt** [reply]
Yes, I checked this part as well. It was "TRUE" . I think there is a hardware deficiency. I heard somewhere that there are 2 horn installations in Megane 4s. The alarm was not ringing due to the deficiency. I think Clio 4 and Captur are the same way. I don't know its accuracy.

**Reply by ntgnt** [reply]
https://www.youtube.com/watch?v=VNPhJC5DxMo&ab_channel=FurkanAyka%C3%A7%2FRetrofitEngineering This was the source. Can be understood by choosing English subtitles
